src/js/components/aboutTheData/AgencyDetailsPage.jsx

```diff
@@ -87,6 +94,11 @@ const AgencyDetailsPage = () => {
                         Agency Submission Data
                     </h1>
                 </div>
+                {agencyOverview?.name && (
+                    <div className="sticky-header__toolbar">
+                        <ShareIcon slug={`${pathname}/${agencyCode}`} email={getEmailDetails(agencyOverview.name)} />
```

Comment:
This results in the agency code twice (e.g. `https://www.usaspending.gov//submission-statistics/agency/024/024`)
```suggestion
                        <ShareIcon slug={`${pathname}`} email={getEmailDetails(agencyOverview.name)} />
```

Reply:
Also just noticed two slashes after .gov 🤔

Reply:
ah thank you!!